api/query/atoms_test.go

```diff
@@ -25,76 +25,76 @@ func TestStructuredQuery_empty(t *testing.T) {
 func TestStructuredQuery_missingRunIDs(t *testing.T) {
 	var rq RunQuery
 	err := json.Unmarshal([]byte(`{
-		"query": {
-			"pattern": "/2dcontext/"
-		}
-	}`), &rq)
+        "query": {
```

Comment:
Did the indention need to change here, or could that be done separately?

Reply:
Let me separate them